PackageSync.sublime-settings

```diff
@@ -21,7 +21,9 @@
 		"Package Control.ca-list",
 		"Package Control.ca-bundle",
 		"Package Control.system-ca-bundle",
-		"*.sublime-package"
+		"*.sublime-package",
+		"PackageSync.sublime-settings",
+		"PackageSync.last-run"
```

Comment:
`PackageSync.last-run` should never be synced as it stores the sync logs. Syncing that would mess up the entire logic & might cause infinite recursion across devices.

Reply:
yeah, I didn't notice the default all files with  suffix `.last-run` will be ignored by default.